scene/main/canvas_item.cpp

```diff
@@ -1136,8 +1136,8 @@ void CanvasItem::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("update"), &CanvasItem::update);
 
-	ClassDB::bind_method(D_METHOD("set_as_toplevel", "enable"), &CanvasItem::set_as_toplevel);
-	ClassDB::bind_method(D_METHOD("is_set_as_toplevel"), &CanvasItem::is_set_as_toplevel);
+	ClassDB::bind_method(D_METHOD("set_as_toplevel", "enable"), &CanvasItem::set_as_top_level);
+	ClassDB::bind_method(D_METHOD("is_set_as_toplevel"), &CanvasItem::is_set_as_top_level);
```

Comment:
You forgot to update those.
```
ERROR: Invalid setter 'CanvasItem::set_as_top_level' for property 'top_level'.
   at: add_property (core/class_db.cpp:985)
```

Reply:
Huh - weird. My bad. [PR](https://github.com/godotengine/godot/pull/42518)